samples/rx-redux/src/main/java/com/pushtorefresh/redom/samples/rxredux/signin/SignInStateMachine.kt

```diff
@@ -27,6 +27,8 @@ class SignInStateMachine(
         abstract val password: CharSequence
         abstract val signInButtonEnabled: Boolean
 
+        // TODO extract state text
```

Comment:
Sho to ya ne ponyal smisla etoi todo

Reply:
pocan k uspehu shel, ne povezlo ne fortanulo